src/test/bridges/angle/Angle.t.sol

```diff
@@ -0,0 +1,273 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Aztec.
+pragma solidity >=0.8.4;
+
+import {BridgeTestBase} from "./../../aztec/base/BridgeTestBase.sol";
+import {AztecTypes} from "../../../aztec/libraries/AztecTypes.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC20Metadata} from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {ICERC20} from "../../../interfaces/compound/ICERC20.sol";
+import {AngleBridge} from "../../../bridges/angle/AngleBridge.sol";
+import {ErrorLib} from "../../../bridges/base/ErrorLib.sol";
+
+import {console} from "forge-std/Test.sol";
+
+contract AngleTest is BridgeTestBase {
+    using SafeERC20 for IERC20;
+
+    AztecTypes.AztecAsset private daiAsset;
+    AztecTypes.AztecAsset private ethAsset;
+    AztecTypes.AztecAsset private wethAsset;
+    AztecTypes.AztecAsset private unsupportedAsset;
+
+    AztecTypes.AztecAsset private sanDaiAsset;
+
+    AngleBridge internal bridge;
+    uint256 internal id;
+
+    uint256 internal constant DUST = 1;
+
+    function setUp() public {
+        emit log_named_uint("block number", block.number);
+        bridge = new AngleBridge(address(ROLLUP_PROCESSOR));
+        vm.label(address(bridge), "ANGLE_BRIDGE");
+        vm.deal(address(bridge), 0);
+
+        vm.startPrank(MULTI_SIG);
+        ROLLUP_PROCESSOR.setSupportedBridge(address(bridge), 5000000);
+        ROLLUP_PROCESSOR.setSupportedAsset(address(0x7B8E89b0cE7BAC2cfEC92A371Da899eA8CBdb450), 1000000);
+        ROLLUP_PROCESSOR.setSupportedAsset(address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2), 1000000);
```

Comment:
Casting the address to address everywhere in the tests is unnecessary but it's just a detail.